pkg/rpc/request/param.go

```diff
@@ -219,11 +238,38 @@ func (p *Param) GetBytesBase64() ([]byte, error) {
 }
 
 // GetSignerWithWitness returns SignerWithWitness value of the parameter.
-func (p Param) GetSignerWithWitness() (SignerWithWitness, error) {
+func (p *Param) GetSignerWithWitness() (SignerWithWitness, error) {
 	c, ok := p.Value.(SignerWithWitness)
+	if ok {
```

Comment:
Isn't then `p.Value` always a raw message?

Reply:
It is set right before the return.